feature/about/src/androidTest/java/io/github/droidkaigi/confsched2019/widget/ExampleInstrumentedTest.java

```diff
@@ -20,6 +20,6 @@ public void useAppContext() {
         // Context of the app under test.
         Context appContext = InstrumentationRegistry.getTargetContext();
 
-        assertEquals("io.github.droidkaigi.confsched2019.ui.test", appContext.getPackageName());
+        assertEquals("io.github.droidkaigi.confsched2019.about.test", appContext.getPackageName());
```

Comment:
Package declaration may be incorrect?
Should I fix AndroidManifest for each module to "io.github.droidkaigi.confsched2019"?

Reply:
Thank you for changing this!
If you change Manifest.xml, it will be an error that BuildCondig is duplicated in the release build, so you should not change it.